drape_frontend/colored_symbol_shape.cpp

```diff
@@ -109,7 +109,11 @@ void ColoredSymbolShape::Draw(ref_ptr<dp::GraphicsContext> context, ref_ptr<dp::
 
   glsl::vec2 const pt = glsl::ToVec2(ConvertToLocal(m_point, m_params.m_tileCenter,
                                                     kShapeCoordScalar));
-  glsl::vec3 const position = glsl::vec3(pt, m_params.m_depth);
+  // Don't pass a real m_params.m_depth value, because shields never overlap
+  // (the displacement engine takes care of this).
+  // But passing it in will make shields out of [dp::kMinDepth, dp::kMaxDepth]
+  // not rendered at all - avoid this so that shields priority range is not restricted.
+  glsl::vec3 const position = glsl::vec3(pt, 0 /* depth */);
```

Comment:
Well, here is a bit strange change, because ColoredSymbolShape may be not only a shield.
I didn't get the problem from the comment, could we make some other fix here?

Reply:
Its basically for all overlays - the depth value is not needed for OGL to render them because they never overlap (due to displacement).
But the depth value is limited by the [kMinDepth;kMaxDepth] range hence limits overlay priorities range unnecessary - e.g. during priorities migration I had it up to 200000 (and kMaxDepth is 25000).

Reply:
So if we don't pass the depth value here then its not getting limited by those bounds.

Reply:
I see its being used for user marks also, I'll double check.

Reply:
For user marks the depth seems to be set to kInvalidDepth always :)

But ok I'll remove this change for now just to play safe. Its not important now as priorities were compressed to fit into a range much smaller than [kMinDepth;kMaxDepth] anyway.